src/test/java/step3/MoneyTest.java

```diff
@@ -0,0 +1,18 @@
+package step3;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import step3.domain.Money;
+
+public class MoneyTest {
+
+    @Test
+    @DisplayName("구입금액이 0미만인 경우 예외가 발생해야 한다.")
+    void moneyIsLessThanZero() {
+        assertThatThrownBy(() -> new Money(-10))
+            .isInstanceOf(IllegalArgumentException.class)
+            .hasMessageContaining(Money.LESS_THAN_MIN_SIZE);
+    }
```

Comment:
현재 테스트도 괜찮지만 [엣지 케이스](https://kathak33.tistory.com/19) 에 대한 테스트도 있으면 좋을 것 같은데 
이 부분에 대해 어떻게 생각하시나요??
현재 테스트명을 참고해본다면 0이 기준이 되므로 
그 경계에 대한 -1 은 생성 실패, 0 은 생성 성공하는 테스트로 두 케이스를 나눠서 구성해보는 방향은 어떠신가요??

Reply:
양수인 경우 금액 객체가 생성되는 테스트 코드를 추가했습니다 .! 혹시 말씀해주신 테스트코드 작성법이 따로 있을까요 ? 제가 잘못이해한 것인지 잘 모르겠어서요!!🤔🤔

Reply:
아래 코드와 같이 엣지 케이스 (Integer.MIN_VALUE, -1) 에 대한 테스트를 말씀드린 거였습니다 ㅎㅎ😆
```suggestion
    @ParameterizedTest
    @ValueSource(ints = {Integer.MIN_VALUE, -10, -1})
    @DisplayName("구입금액이 0미만인 경우 예외가 발생해야 한다.")
    void moneyIsLessThanZero(int value) {
        assertThatThrownBy(() -> new Money(value))
            .isInstanceOf(IllegalArgumentException.class)
            .hasMessageContaining(Money.LESS_THAN_MIN_SIZE);
    }
```